resources/allowlist.gni

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Allowlist file used by brave resources, created after merging chrome allowlist
+# and brave resources.
+brave_allowlist_file = "${root_gen_dir}/brave_resources_allowlist.txt"
```

Comment:
I'm confused, what generates this file?

Reply:
I see now, but this is very confusing to put it here. The merge allowlist target should be moved out of wireguard and probably into `brave/BUILD.gn`

Reply:
moved